modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -67,6 +72,10 @@ export default class Ecdsa {
     // Add secret shares
     const x = allShares.map((participant) => BigInt(participant['u'])).reduce(Ecdsa.curve.scalarAdd);
 
+    // Chaincode will be used in future when we add support for key derivation for ecdsa
+    const chaincodes = [pShare, ...nShares].map(({ chaincode }) => bigIntFromBufferBE(Buffer.from(chaincode, 'hex')));
+    const chaincode = chaincodes.reduce((acc, chaincode) => Ecdsa.curve.scalarReduce(acc + chaincode));
```

Comment:
The chaincode isn't an element in the secp256k1 field. The common chaincode should be calculated as `(acc + chaincode) % (1n<<256n)`